README.md

```diff
@@ -1,6 +1,8 @@
 # DistributionsAD.jl
 
-[![ForwardDiff and Tracker tests](https://github.com/TuringLang/DistributionsAD.jl/workflows/ForwardDiff%20and%20Tracker%20tests/badge.svg?branch=master)](https://github.com/TuringLang/DistributionsAD.jl/actions?query=workflow%3A%22ForwardDiff+and+Tracker+tests%22)
+![Zygote tests](https://github.com/TuringLang/DistributionsAD.jl/workflows/Zygote%20tests/badge.svg)
```

Comment:
A slightly different badge for Zygote tests is already included in the README below. It explicitly only displays the results of the latest CI run on the master branch (the same is done for all other badges as well).